single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -1206,7 +1195,7 @@ private void updateLayoutsFactory(Network network) {
     }
 
     private void initStylesProvider() {
-        styles.put("Default", new DefaultDiagramStyleProvider());
+        styles.put("Default", new BasicStyleProvider());
```

Comment:
As it's not the default style provider anymore, you could rename the text to "Basic"